PR: https://github.com/odoo/odoo/pull/178200

From: f10bd8ddcf99c9247dca417fefbc260454d74ea1
From: Gorash
Date: 2024-10-15 09:33:02

Breaking data model changes scores: del:2 + add:0, change matches:
-    _inherit = 'account.move'
-    _inherit = 'res.partner'

Total Changes: 52

[IMP] *: adapt model class names to correspond to model names (apply script)

Part-of: odoo/odoo#178200
Related: odoo/enterprise#69762
Related: odoo/documentation#11115
Related: odoo/upgrade#6577
Related: odoo/design-themes#988
Related: odoo/upgrade-util#143

================================= pseudo patch: =================================

--- a/addons/account_edi_ubl_cii/models/account_edi_common.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_common.py
@@ -92,7 +92,6 @@ EAS_MAPPING = {
 
 
 class AccountEdiCommon(models.AbstractModel):
-    _name = "account.edi.common"
     _description = "Common functions for EDI documents: generate the data, the constraints, etc"
 
     # -------------------------------------------------------------------------

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py
@@ -16,9 +16,8 @@ CII_NAMESPACES = {
 }
 
 
-class AccountEdiXmlCII(models.AbstractModel):
-    _name = "account.edi.xml.cii"
-    _inherit = 'account.edi.common'
+class AccountEdiXmlCii(models.AbstractModel):
+    _inherit = ['account.edi.common']
     _description = "Factur-x/XRechnung CII 2.2.0"
 
     def _find_value(self, xpath, tree, nsmap=False):

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py
@@ -10,9 +10,8 @@ UBL_NAMESPACES = {
 }
 
 
-class AccountEdiXmlUBL20(models.AbstractModel):
-    _name = "account.edi.xml.ubl_20"
-    _inherit = 'account.edi.common'
+class AccountEdiXmlUbl_20(models.AbstractModel):
+    _inherit = ['account.edi.common']
     _description = "UBL 2.0"
 
     def _find_value(self, xpath, tree, nsmap=False):

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_21.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_21.py
@@ -2,9 +2,8 @@
 from odoo import api, models
 
 
-class AccountEdiXmlUBL21(models.AbstractModel):
-    _name = "account.edi.xml.ubl_21"
-    _inherit = 'account.edi.xml.ubl_20'
+class AccountEdiXmlUbl_21(models.AbstractModel):
+    _inherit = ['account.edi.xml.ubl_20']
     _description = "UBL 2.1"
 
     # -------------------------------------------------------------------------

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_a_nz.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_a_nz.py
@@ -3,9 +3,8 @@
 from odoo import models
 
 
-class AccountEdiXmlUBLANZ(models.AbstractModel):
-    _inherit = "account.edi.xml.ubl_bis3"
-    _name = 'account.edi.xml.ubl_a_nz'
+class AccountEdiXmlUbl_A_Nz(models.AbstractModel):
+    _inherit = ["account.edi.xml.ubl_bis3"]
     _description = "A-NZ BIS Billing 3.0"
 
     """

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py
@@ -6,9 +6,8 @@ from odoo.addons.account_edi_ubl_cii.models.account_edi_xml_ubl_20 import UBL_NA
 from stdnum.no import mva
 
 
-class AccountEdiXmlUBLBIS3(models.AbstractModel):
-    _name = "account.edi.xml.ubl_bis3"
-    _inherit = 'account.edi.xml.ubl_21'
+class AccountEdiXmlUbl_Bis3(models.AbstractModel):
+    _inherit = ['account.edi.xml.ubl_21']
     _description = "UBL BIS Billing 3.0.12"
 
     """

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_efff.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_efff.py
@@ -5,9 +5,8 @@ from odoo import models
 import re
 
 
-class AccountEdiXmlUBLEFFF(models.AbstractModel):
-    _inherit = "account.edi.xml.ubl_20"
-    _name = 'account.edi.xml.ubl_efff'
+class AccountEdiXmlUbl_Efff(models.AbstractModel):
+    _inherit = ["account.edi.xml.ubl_20"]
     _description = "E-FFF (BE)"
 
     # -------------------------------------------------------------------------

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_nlcius.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_nlcius.py
@@ -3,9 +3,8 @@
 from odoo import models
 
 
-class AccountEdiXmlUBLNL(models.AbstractModel):
-    _inherit = "account.edi.xml.ubl_bis3"
-    _name = 'account.edi.xml.ubl_nl'
+class AccountEdiXmlUbl_Nl(models.AbstractModel):
+    _inherit = ["account.edi.xml.ubl_bis3"]
     _description = "SI-UBL 2.0 (NLCIUS)"
 
     """

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_sg.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_sg.py
@@ -2,9 +2,8 @@
 from odoo import models
 
 
-class AccountEdiXmlUBLSG(models.AbstractModel):
-    _inherit = "account.edi.xml.ubl_bis3"
-    _name = "account.edi.xml.ubl_sg"
+class AccountEdiXmlUbl_Sg(models.AbstractModel):
+    _inherit = ["account.edi.xml.ubl_bis3"]
     _description = "SG BIS Billing 3.0"
 
     """

--- a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_xrechnung.py
+++ b/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_xrechnung.py
@@ -2,9 +2,8 @@
 from odoo import models
 
 
-class AccountEdiXmlUBLDE(models.AbstractModel):
-    _inherit = "account.edi.xml.ubl_bis3"
-    _name = 'account.edi.xml.ubl_de'
+class AccountEdiXmlUbl_De(models.AbstractModel):
+    _inherit = ["account.edi.xml.ubl_bis3"]
     _description = "BIS3 DE (XRechnung)"
 
     # -------------------------------------------------------------------------

--- a/addons/account_edi_ubl_cii/models/account_move.py
+++ b/addons/account_edi_ubl_cii/models/account_move.py
@@ -2,7 +2,7 @@ from odoo import _, api, fields, models, Command
 
 
 class AccountMove(models.Model):
-    _inherit = 'account.move'
+    _inherit = ['account.move']
 
     ubl_cii_xml_id = fields.Many2one(
         comodel_name='ir.attachment',

--- a/addons/account_edi_ubl_cii/models/account_move_send.py
+++ b/addons/account_edi_ubl_cii/models/account_move_send.py
@@ -13,7 +13,7 @@ _logger = logging.getLogger(__name__)
 
 
 class AccountMoveSend(models.AbstractModel):
-    _inherit = 'account.move.send'
+    _inherit = ['account.move.send']
 
     # -------------------------------------------------------------------------
     # ALERTS

--- a/addons/account_edi_ubl_cii/models/res_partner.py
+++ b/addons/account_edi_ubl_cii/models/res_partner.py
@@ -8,7 +8,7 @@ from odoo.addons.account_edi_ubl_cii.models.account_edi_common import EAS_MAPPIN
 
 
 class ResPartner(models.Model):
-    _inherit = 'res.partner'
+    _inherit = ['res.partner']
 
     invoice_edi_format = fields.Selection(
         selection_add=[
